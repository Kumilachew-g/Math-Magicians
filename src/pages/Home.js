import React from 'react';
import ParticlesComponent from 'react-particles-js';

const Home = () => (
  <div>
    <div className='welcome'>
      <h1>Welcome to Our Page!</h1>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni maiores
        nisi consectetur autem ullam sapiente, quaerat ex aperiam in, veritatis
        culpa ab aliquam dolore odit! Reiciendis labore magni excepturi odio?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni maiores
        nisi consectetur autem ullam sapiente, quaerat ex aperiam in, veritatis
        culpa ab aliquam dolore odit! Reiciendis labore magni excepturi odio?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni maiores
        nisi consectetur autem ullam sapiente, quaerat ex aperiam in, veritatis
        culpa ab aliquam dolore odit! Reiciendis labore magni excepturi odio?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni maiores
        nisi consectetur autem ullam sapiente, quaerat ex aperiam in, veritatis
        culpa ab aliquam dolore odit! Reiciendis labore magni excepturi odio?
      </p>
    </div>
    <ParticlesComponent />
  </div>
);

export default Home;
