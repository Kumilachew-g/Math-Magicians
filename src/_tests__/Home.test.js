import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/Home';

it('expect homepage to match working snapshot', () => {
  const homePage = renderer.create(<Home />).toJSON();
  expect(homePage).toMatchSnapshot();
});
