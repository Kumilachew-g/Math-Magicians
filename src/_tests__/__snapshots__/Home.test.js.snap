// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect homepage to match working snapshot 1`] = `
<div>
  <div
    className="welcome-home"
  >
    <h1>
      Welcome to Our Page
    </h1>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit, laborum, odio quis rerum fugiat reprehenderit numquam nisi aperiam non, ab blanditiis quisquam eum fugit dicta illo soluta facere iure mollitia?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit, laborum, odio quis rerum fugiat reprehenderit numquam nisi aperiam non, ab blanditiis quisquam eum fugit dicta illo soluta facere iure mollitia?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit, laborum, odio quis rerum fugiat reprehenderit numquam nisi aperiam non, ab blanditiis quisquam eum fugit dicta illo soluta facere iure mollitia?
    </p>
    <p>
      Lorem ipsum dolor, sit amet consectetur adipisicing elit. Odit, laborum, odio quis rerum fugiat reprehenderit numquam nisi aperiam non, ab blanditiis quisquam eum fugit dicta illo soluta facere iure mollitia?
    </p>
  </div>
  <div
    id="tsparticles"
  >
    <canvas
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
